test/dlc_tests/container_tests/bin/test_efa_sanity.py

```diff
@@ -0,0 +1,62 @@
+import subprocess
+import sys
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+def main():
+    check_ib_uverbs()
+    check_fi_info()
+    check_efa_installed_packages()
+    run_fi_pingpong_test()
+
+    return 0
+
+def check_ib_uverbs():
```

Comment:
Why do all the functions in this script `return True` when the return value is untested?

Reply:
yup. dropping the True.